docs/site/content/docs/tmc-register.md

```diff
@@ -0,0 +1,53 @@
+# Register a Management Cluster in Tanzu Mission Control
+
```

Comment:
Let's add a short description here of what TMC is.

Namely letting folks know that TMC is a VMware *product* (not free/open source project)